docs/plugins/channel-safely.rst

```diff
@@ -0,0 +1,66 @@
+channel-safely
+==============
+
+.. dfhack-tool::
+    :summary: Auto-manage channel designations to keep dwarves safe
+    :tags: fort auto
+
+Multi-level channel projects can be dangerous, and managing the safety of your
+dwarves throughout the completion of such projects can be difficult and time
+consuming. This plugin keeps your dwarves safe (while channeling) so you don't
+have to. Now you can focus on designing your dwarven cities with the deep chasms
+they were meant to have.
+
+Usage
+-----
+
+::
+    enable channel-safely
+    channel-safely set <setting> <value>
+    channel-safely enable|disable <feature>
+    channel-safely runonce
+
+When enabled the map will be scanned for channel designations which will be grouped
+together based on adjacency and z-level. These groups will then be analyzed for safety
+and designations deemed unsafe will be put into :wiki:`Marker Mode <Designations_menu#Marker_Mode>`.
+Each time a channel designation is completed its group status is checked, and if the group
+is complete pending groups below are made active again.
+
+Examples
+--------
+
+``channel-safely``
+    The plugin reports its configured status.
+
+``channel-safely runonce``
+    Runs the safety procedures once. You can use this if you prefer initiating scans manually.
+
+``channel-safely disable require-vision``
+    Allows the plugin to read all tiles, including the ones your dwarves know nothing about.
+
+``channel-safely enable monitor-active``
+    Enables monitoring active channel digging jobs. Meaning that if another unit it present
+    or the tile below becomes open space the job will be paused or canceled (respectively).
+
+``channel-safely set ignore-threshold 3``
+    Configures the plugin to ignore designations equal to or above priority 3 designations.
+
+Features
+--------
+
+:require-vision:    Toggle whether the dwarves need vision of a tile before channeling to it can be deemed unsafe. (default: enabled)
+:monitor:           Toggle whether to monitor the conditions of active digs. (default: disabled)
```

Comment:
```suggestion
``channel-safely enable monitor``
    Enables monitoring active channel digging jobs. Meaning that if another unit it present
    or the tile below becomes open space the job will be paused or canceled (respectively).

``channel-safely set ignore-threshold 3``
    Configures the plugin to ignore designations equal to or above priority 3 designations.

Features
--------

:require-vision:    Toggle whether the dwarves need vision of a tile before channeling to it can be deemed unsafe. (default: enabled)
:monitor:           Toggle whether to monitor the conditions of active digs. (default: disabled)
```